ribs/archives/_cvt_archive.py

```diff
@@ -118,6 +118,8 @@ def __init__(self,
                  dtype=np.float64,
                  samples=100_000,
                  custom_centroids=None,
+                 chunk_size=10,
+                 chunk_distances=False,
```

Comment:
I think we don't need this argument. We can define the behavior of chunk computation as follows:
- `chunk_size` defaults to `np.inf`.
- When `chuck_size < measures_batch.shape[0]`, we perform chunk computation; otherwise, we compute everything at the same time.

Reply:
Chunk_size defaults to np.inf now. Right now the code defaults to a single computation if chunk_size < measures_batch.shape[0], I can still add that check though for clarity

Reply:
`chunk_size` should default to None. `np.inf` is a floating value, while `chunk_size` must be an int. Since int's have no special indicator for inf, we can use None instead. (semantically, "None" would mean "no chunking")